src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -2368,7 +2368,7 @@ export default {
 		checkFailed: 'Check failed',
 		openBackofficeSearch: 'Open backoffice search',
 		openCloseBackofficeHelp: 'Open/Close backoffice help',
-		openCloseBackofficeProfileOptions: 'Open/Close your profile options',
+		BackofficeProfileOptions: 'Profile options',
```

Comment:
FYI, localization keys are generally in camelCase. We can either do a lowercase `b`, or maybe even simplify the key to be `profileOptions`?